interview-questions.md

```diff
@@ -7,43 +7,33 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
-
-  Researched answer:
-
+  Your answer: An instance variable is a variable that starts with @ symbol and belongs to object, wheres class variables belong only to the class.
 
+  Researched answer: An instance variable starts with a @ symbol as well initialized to the value of nil. Instance variables belong to the object itself. They can be accessed throughout the class as well.
```

Comment:
Yea, the instance variable is a value that is available to the whole instance object.